### ISSUE TITLE
1.10 Version

### ISSUE THREAD

**Issue by ElementalRealm** [PEXPlugins/Modifyworld#118]
Is there any way to get the 1.10 minecraft version of this plugin?
Thanks.

- Elemtental Realm

**Comment by Pieter12345** [comment]
Yes, you can re-add the following method to org.bukkit.configuration.file.YamlConfiguration:

`     /**
     * Creates a new {@link YamlConfiguration}, loading from the given stream using UTF-8 encoding.
     * <p>
     * Any errors loading the Configuration will be logged and then ignored.
     * If the specified input is not a valid config, a blank config will be
     * returned.
     *
     * @param stream Input stream
     * @return Resulting configuration
     * @throws IllegalArgumentException Thrown if stream is null
     * @deprecated This method is no longer part of the official Bukkit.
     *  It has been added by woesh0007 for compatibility.
     * @see #load(InputStream)
     * @see #loadConfiguration(Reader)
     */
    @Deprecated
    public static YamlConfiguration loadConfiguration(InputStream stream) {
        Validate.notNull(stream, "Stream cannot be null");
        InputStreamReader reader;
		try {
			reader = new InputStreamReader(stream, "UTF-8");
		} catch (UnsupportedEncodingException e) {
			throw new RuntimeException(e); // Never happens.
		}
        return loadConfiguration(reader);
    }`
